packages/rpm/hazelcast.spec

```diff
@@ -1,11 +1,13 @@
 %define hzversion ${HZ_VERSION}
+%define rpmhzversion ${RPM_HZ_VERSION}
```

Comment:
Why do we need this define?

Reply:
We use it in:
```
Version:    %{rpmhzversion}
```
where the version is _upstream_ version (= version of hazelcast we package), but in rpm format - no hyphens, e.g.: hzversion=5.1-BETA-1, rpmversion=5.1.BETA.1
It could be also done via the envsubst but since we have the hzversion as well I left it there.